src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -466,6 +503,11 @@ private void updateBundleAndRecipientListWithResponsesForStudent(
 
         bundle.put(question, responses);
         recipientList.put(question.getId(), recipients);
+        for (FeedbackResponseAttributes response : responses) {
+            List<FeedbackResponseCommentAttributes> comments =
+                    frcLogic.getFeedbackResponseCommentForResponse(response.getId());
+            commentsForResponses.put(response.getId(), comments);
```

Comment:
Here I haven't checked if `comments` is empty. Should I? It is beneficial sometimes as it doesn't throw null pointer exception.